incubator/redis-cache/values.yaml

```diff
@@ -0,0 +1,46 @@
+# Default values for redis-cache.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+##Statefulset related configurations ss.yaml
+replicaCount: 3
```

Comment:
Im not sure this is entirely wrong we have plenty of charts using this already, even recently accepted ones? 

```
$find . -name "*.yaml" | grep -v incubator | xargs grep -n replicaCount | wc -l
37
```